src/listeners/services/youtube/readyNotificationStream.ts

```diff
@@ -0,0 +1,15 @@
+import { envParseBoolean } from '#lib/env';
+import { Events } from '#lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Listener, ListenerOptions } from '@sapphire/framework';
+
+@ApplyOptions<ListenerOptions>({ event: 'ready', once: true, enabled: envParseBoolean('GRPC_NOTIFICATIONS_ENABLED') })
+export class UserListener extends Listener {
+	public async run() {
+		const { client, grpc } = this.container;
+
+		for await (const notification of grpc.youtube.getNotificationStream()) {
```

Comment:
if the microservice is ever restarted; will this throw? If so, maybe a good idea to handle reconnection logic here.

Reply:
I... have no idea, I can't run this service locally to find out.